src/app/services/views/ServiceListView.vue

```diff
@@ -65,14 +85,7 @@
               </template>
 
               <template #status="{ row: item }">
-                <StatusBadge
-                  v-if="item.status"
-                  :status="item.status"
-                />
-
-                <template v-else>
-                  —
-                </template>
+                <StatusBadge :status="item.status || 'not_available'" />
```

Comment:
Would it make sense to have this done in your `transfromToTableData` ?

Reply:
Technically, yes, but it looks very much like this (these transform call) is all going to be thrown out in favor of a different approach.

Reply:
Yes but the transform call will likely be the base for the code of the new approach no?